src/main/kotlin/uk/gov/dluhc/printapi/config/StringToAedSearchByConverter.kt

```diff
@@ -0,0 +1,24 @@
+package uk.gov.dluhc.printapi.config
+
+import mu.KotlinLogging
+import org.springframework.core.convert.converter.Converter
+import org.springframework.stereotype.Component
+import uk.gov.dluhc.printapi.models.AedSearchBy
+
+private val logger = KotlinLogging.logger {}
+
+/**
+ * [Converter] that converts a String to [AedSearchBy], supporting Spring binding of web request parameters
+ * to [SearchBy] variables
+ */
+@Component
+class StringToAedSearchByConverter : Converter<String, AedSearchBy> {
```

Comment:
My tests were failing due to absence of this String to enum converter 🤦🏼‍♂️ . 
I thought the `QueryStringParameterPropertyEditor` in Controller was enough.